bootstrap/terraform/aws-bootstrap/output.tf

```diff
@@ -32,5 +32,5 @@ output "cluster_public_subnet_ids" {
 }
 
 output "node_groups" {
```

Comment:
could we also grab the first groups worker_role_arn to make life easier for downstream apps?